concent_api/api_testing_common.py

```diff
@@ -455,6 +456,16 @@ def create_signed_report_computed_task(
                 task_to_compute=task_to_compute,
                 size=REPORT_COMPUTED_TASK_SIZE,
             ),
-            PROVIDER_PRIVATE_KEY,
+            provider_private_key,
         )
         return signed_message
+
+
+def get_cluster_name_from_address(address):
```

Comment:
I think it should be part of `SCIBaseTest`.

Reply:
Fixed